.circleci/config.yml

```diff
@@ -55,13 +55,13 @@ jobs:
       - run: yarn run wait-on http://localhost:6060 && yarn cypress run
   deploy:
     docker:
-      - image: circleci/node:8.11.1
+      - image: circleci/node:10.18.0
     working_directory: ~/repo
     steps:
       - checkout
       - install_dependencies:
           image: "node"
-          version: "1"
+          version: "10.18.0"
```

Comment:
Holy wow, were we really using node v1?! 😮

Reply:
Oh wait, it seems like this is redundant, considering the image itself has node already installed? Can we remove the dependency completely?

Reply:
We were using Node 8! The version value here is just for the cache key. I decided to just keep it the same as the actual Node version for the sake of consistency (and easier understanding in the future of when we need to bust the key).